examples/tv-casting-app/linux/simple-app.cpp

```diff
@@ -148,6 +174,16 @@ int main(int argc, char * argv[])
     VerifyOrReturnValue(err == CHIP_NO_ERROR, 0,
                         ChipLogError(AppServer, "CastingApp::Start failed %" CHIP_ERROR_FORMAT, err.Format()));
 
+    DiscoveryDelegateImpl delegate;
+    CastingPlayerDiscovery::GetInstance()->SetDelegate(&delegate);
+    VerifyOrReturnValue(err == CHIP_NO_ERROR, 0,
+                        ChipLogError(AppServer, "CastingPlayerDiscovery::SetDelegate failed %" CHIP_ERROR_FORMAT, err.Format()));
+
+    // Discover CastingPlayers
+    err = CastingPlayerDiscovery::GetInstance()->StartDiscovery(35); // 35 represents device type: Matter video players
```

Comment:
Make the device type 35 a constant (kTargetPlayerDeviceType) in this file.

Reply:
added!